src/lib/components/EventNetwork.svelte

```diff
@@ -294,20 +445,20 @@
         const y = bounds.y;
 
         // Calculate scale to fit
-        const scale = 1.25 / Math.max(dx / width, dy / height);
-        const translate = [
-          width / 2 - scale * (x + dx / 2),
-          height / 2 - scale * (y + dy / 2),
-        ];
+        // const scale = 1.25 / Math.max(dx / width, dy / height);
```

Comment:
If we don't plan to restore this commented-out code, we should remove it entirely.